docs/cc_to_ako.md

```diff
@@ -0,0 +1,89 @@
+
+## Workflow
+
+This section gives details about steps of migration.
+
+
+![Alt text](images/workflow-cc-to-ako.png?raw=true)
+
+### 1. Preparation for migration
+
+*  Setup a new controller (compatible with AKO version 1.4.1 and above)
+*  Setup a new Vcenter cloud with IPAM.
+*  Setup a DNS Service
+*  Replicate AVI side objects referred by Ingresses/Services as part of AVI_PROXY annotations to new controller.
+
+### 2. Deploy AKO CRDs
+
+For each AVI_PROXY annotation present in an ingress, create corresponding AKO Http rule/Host rule/ AviInfrasetting rule.
+Details of these CRDs can be found out [here](crds/overview.md)
+
+If any AVI_PROXY annotation is not supported by these CRD's, then VS which is migrated to AKO will not have same features as that in CloudConnector. 
+
+### 3. AKO Deployment
+
+Deploy AKO (version 1.4 and above) with namespace sync feature enabled.
+
+![Alt text](images/ako-namespace-selector.png)
+
+As an example, AKO is deployed with `app` as a key and `migrate` as a value for namespace selector. So AKO will sync up all objects from namespace(s) with this label and corresponding Avi objects will be created in Avi-Controller.
+
+### 4. Namespace-Driven Exclusion of OpenShift/Kubernetes Applications
+
+Set an exclude attribute for cloud connector cloud (Openshift Cloud) either from UI or AVI shell. Key and value should be same as mentioned in AKO.
+
+* Setting up exclusion attribute using UI
+![Alt text](images/cloud-connector-gui-screenshot.png)
+
+* Setting up exclusion attribute using Avi shell
+![Alt text](images/cloud-connector-shell-screenshot.png)
+
+As an example, exclusion attribute as `app` and exclusion value as `migrate` is used. This will delete virtual services of namespace(s), which has same label, from CloudConnector Avi-Controller.
+
+### 5. Kubernetes namespace label
+
+Label namespace(s) in kubernetes cluster with same `key:value` pair.
+
+![Alt text](images/k8-namespace-labels.png)
+
+As shown in above snippet, "red" namespace is labelled with "app: migrate". This will result in deleting virtual services of red namespace from CC Avi-controller and creating new virtual services for red namespace in AKO Avi-controller. There will be traffic disruption during migration.
+
+### 6. Internal testing
+
+When objects migrates to new AKO Avi controller, it acquires new VIP. With that new VIP, do internal traffic test for migrated applications.
+
+After successful testing, objects from remaining namespaces can be migrated by labelling namespace with valid label (as done in step 5).
+
+### 7. Client traffic redirection
+
+Once all ingresses, L4 services migrated to AKO Avi controller, client traffic can be redirected to new Vips by
+* either changing DNS server entry in corporate DNS server to point out to new DNS service from AKO based Vcenter Cloud
```

Comment:
Do you need a bullet here? The sentence seems to start with a small letter.

Change the DNS server entry in corporate DNS server to point to the new DNS service on the AKO based Avi Controller.